Makefile

```diff
@@ -2,7 +2,7 @@ DIST_DIR = ./dist/
 DIST := skygear-server
 VERSION := $(shell git describe --always --tags)
 GO_BUILD_LDFLAGS := -ldflags "-X github.com/skygeario/skygear-server/pkg/server/skyversion.version=$(VERSION)"
-GO_TEST_TIMEOUT := 1m
+GO_TEST_TIMEOUT := 1m30s
```

Comment:
The commit message says to see if the test would pass if the timeout is extended. Is this still needed after you finished your changes?